pkg/api/admin/openshiftcluster.go

```diff
@@ -24,6 +24,7 @@ type OpenShiftCluster struct {
 	Location   string                     `json:"location,omitempty"`
 	Tags       map[string]string          `json:"tags,omitempty"`
 	Properties OpenShiftClusterProperties `json:"properties,omitempty"`
+	SystemData SystemData                 `json:"systemData,omitempty"`
```

Comment:
I think this is fine for now. Data should be there for old cluster. LastUpdate one minimum.

Reply:
@mjudeikis as far as I understand this is not quite right. We will not have last update fields in the begining. We will only have it after receiving first PUT or PATCH request from ARM. So some old clusters which customers do not modify (from ARM perspective) will be returning `"systemData":{}`.

I think this is fine. Just want to double check.

Reply:
Where do they will return it? Swagger does not have this field in old API, nor in our side old APIS?
This will get returned only if customer interact with old cluster using new api. And for this I think this is fine to have empty values.

Reply:
> This will get returned only if customer interact with old cluster using new api. And for this I think this is fine to have empty values.

Yes, this is exactly what I mean: `GET`ing an old cluster via new API before making any `PUT`s or `PATCH`es.

I also think it is fine. Again: just wanted to double check.